gtdynamics/universal_robot/Link.h

```diff
@@ -121,14 +117,8 @@ class Link : public boost::enable_shared_from_this<Link> {
   /// add joint to the link
   void addJoint(const JointSharedPtr &joint) { joints_.push_back(joint); }
 
-  /// transform from link to world frame
-  const gtsam::Pose3 &wTl() const { return wTl_; }
-
-  /// transform from link CoM frame to link frame
-  const gtsam::Pose3 &lTcom() const { return lTcom_; }
-
-  /// transform from link CoM frame to world frame
-  inline const gtsam::Pose3 wTcom() const { return wTl() * lTcom(); }
+  /// transform from base frame to link COM frame
```

Comment:
`/// Relative pose at rest from link’s COM to the base frame.`

Reply:
done